scipy/spatial/_plotutils.py

```diff
@@ -153,6 +153,32 @@ def voronoi_plot_2d(vor, ax=None, **kw):
     -----
     Requires Matplotlib.
 
+    Examples
+    --------
+    Set of point:
+
+    >>> import numpy as np    
```

Comment:
This line isn't needed, this import is implicit for every single example within scipy.